.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
```

Comment:
For opinionated teams I don't see a problem in choosing your own style guides as long as they are agreed with the rest of the team, however I want everyone to start at the same point and make changes if they really feel they don't agree.  If there is no starting point every single problem is completely different.